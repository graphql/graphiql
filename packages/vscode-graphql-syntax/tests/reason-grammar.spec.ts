import { tokenizeFile } from './__utilities__/utilities';

describe('inline.graphql.reason grammar', () => {
  const scope = 'inline.graphql.re';

  it('should tokenize a simple reasonML file', async () => {
    const result = await tokenizeFile('__fixtures__/test.re', scope);
    expect(result).toMatchSnapshot();
  });
});
