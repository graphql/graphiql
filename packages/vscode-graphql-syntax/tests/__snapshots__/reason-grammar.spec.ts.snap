// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`inline.graphql.reason grammar > should tokenize a simple reasonML file 1`] = `
module VideoGames =                                            | 
[%graphql                                                      | 
  {|                                                           | meta.embedded.block.graphql
                                                               | meta.embedded.block.graphql
query                                                          | meta.embedded.block.graphql keyword.operation.graphql
                                                               | meta.embedded.block.graphql
VideoGames                                                     | meta.embedded.block.graphql entity.name.function.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql
{                                                              | meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql
videoGames                                                     | meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
{                                                              | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql punctuation.operation.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
id                                                             | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql variable.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
title                                                          | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql variable.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
developer                                                      | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql variable.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
completed                                                      | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql variable.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql
}                                                              | meta.embedded.block.graphql meta.selectionset.graphql meta.selectionset.graphql punctuation.operation.graphql
                                                               | meta.embedded.block.graphql meta.selectionset.graphql
}                                                              | meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
|}                                                             | meta.embedded.block.graphql
]                                                              | meta.embedded.block.graphql
;                                                              | 
                                                               | 
module VideoGamesQuery = ReasonApollo.CreateQuery(VideoGames); | 
                                                               | 
`;
